scala-native-jdbc-duckdb-tests/src/test/scala/tests/SimpleTest.scala

```diff
@@ -0,0 +1,168 @@
+package tests
+
+import java.sql.*
+
+class SimpleTest extends munit.FunSuite {
```

Comment:
Note: This is not just a copy of the SQLite tests:

- The Id is explicitly passed on `INSERT`
- orders.total was changed to `DOUBLE` instead of `DECIMAL`